### PR TITLE
Complaint tool benefits info

### PR DESCRIPTION
**Pull request by rrmatthews**
These changes set up the applicant information page for the complaint tool as described by this [ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11575)

## TODO:
- [x] Write schema / uiSchema
- [x] Write unit tests
- [ ] Verify content

## Notes: 
- The "(select all that apply)" isn't grey / not sure how to make that change if the color is important

## Screenshots: 
localhost_3001_education_complaint-tool_form_feedback-information

### CODE REVIEW

File: src/applications/edu-benefits/complaint-tool/config/form.js
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/config/form.js]: It looks consistent with how the other forms are set up to set showFieldLabel to true.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu thanks.. Please see my edits below. Let me know if you have any questions. 

We need to replace "Complaint" with "Feedback" in the breadcrumbs

it looks like we're missing (Select all that apply) from the first two questions. 

Also, the content/list for the first two questions "Which education benefits have you used?" and "Which military tuition assistance have you used?" isn't the correct and doesn't reflect the up-to-date content, per the prototype.

**Comment by erikphansen** [comment]
It looks like this branch has a ton of commits that have already been merged into `master`. Is that going to get cleaned up before this PR is merged?

**Comment by cehsu** [comment]
Good question @erikphansen , this branch has changes from the "updated headers and subway page" PR (#8018), and once it is merged, the changes in this PR will only relate to the form fields.

**Comment by rtluu** [comment]
Is there a reason that the first question 'Which Education Benefits have you used?' isn't bolded like the others?

File: src/applications/edu-benefits/complaint-tool/config/form.js
Comment (rrmatthews) [reply src/applications/edu-benefits/complaint-tool/config/form.js]: @cehsu I updated the screenshot to show you what it looks like if we include it.

**Comment by rrmatthews** [comment]
@peggygannon Changes have been made and screenshot has been updated. 

-I've updated the breadcrumb in the other PR (when we merge the pages together, the breadcrumb and other headers will be correct everywhere)
-The "(Select all that apply)" text isn't grey but I am looking into it to see if this is possible to change

**Comment by rianfowler** [comment]
I'm not able to get through the applicant information page to test this page when running locally. The first email address field is blocking input.

**Comment by rrmatthews** [comment]
@rianfowler I believe it's like that for the other pages as well but that wasn't intentional... I usually just select anonymous and continue on to the next page, though I've been able to just use http://localhost:3001/education/complaint-tool/form/benefits-information to navigate to the page

**Comment by rianfowler** [comment]
@rrmatthews It would help if I was on the right branch too 🤦‍♂️ 

@cehsu not sure how we usually handle these kinds of values- do we want to default false so the review page doesn't show empty  answers on this list? 

screen shot 2018-07-26 at 09 45 56

**Comment by cehsu** [comment]
@rianfowler good idea about defaults, I'll add those in the backend schema.

**Comment by rrmatthews** [comment]
@peggygannon changes have been made and screenshots have been updated

**Review by cehsu — APPROVED** [review]
LGTM, pending @peggygannon's approval. We will want to merge this after the applicant information page PR, since this PR includes those changes (I have updated the applicant information changes to be in sync with the latest version of that PR). I removed the contact information changes, so that the contact information PR would not block this one.

**Comment by peggygannon** [comment]
@cehsu @rrmatthews I don't see an updated screenshot with changes incorporated.

**Comment by rrmatthews** [comment]
@peggygannon sorry, screenshot is updated!

**Comment by peggygannon** [comment]
@rrmatthews @cehsu The titles of the below benefits are not correct and don't match what's in the prototype. Can you please drop in these correct title of these programs? 

Survivors’ and Dependents’ Assistance (DEA) (Chapter 35)
State-funded Tuition Assistance (TA) for Servicemembers on Active Guard and Reserve (AGR) duties

**Comment by cehsu** [comment]
Thanks @peggygannon, kindly refer to the updated screenshot:
localhost_3001_education_complaint-tool_form_applicant-information

**Review by peggygannon — APPROVED** [review]
@cehsu Complaint still needs to be removed from the breadcrumbs... Otherwise copy looks good.

**Comment by cehsu** [comment]
Thanks again @peggygannon, kindly refer to the updated screenshot:
localhost_3001_education_complaint-tool_form_applicant-information 2

**Pull request merged**
